open-craft/frontend-app-course-authoring
# feat: library home page bare bones (TEMP)

## Description
This PR adds a "Bare Bones" version of the Library Home and updates the SearchResult component to redirect to this new page.

![image](https://github.com/open-craft/frontend-app-course-authoring/assets/849463/0611fdfa-e357-471d-9561-7ea0ee0a696a)

![image](https://github.com/open-craft/frontend-app-course-authoring/assets/849463/e1bf1807-6a02-47e3-a268-38d210897687)

## More Info
- Closes: https://github.com/openedx/frontend-app-course-authoring/issues/1036
- Depends on: https://github.com/openedx/frontend-app-course-authoring/pull/1050

## Testing Instructions
- Run your local stack on this branch
- Make sure you have the studio.library_authoring_mfe waffle flag **DISABLED**
- Go to the Studio Home and check the "Libraries" tab
- Click on one library with components
- Check if the component count is shown in the last card and the "Components" tab
- Use the keyword search box and check if the component count is filtered accordingly
- Navigate through the Home, Components and Collections tab and check if you can navigate back using the "Back" button from the brownser
- Go back to the Studio Home and open a library WITHOUT components
- You should see a message with a button that will allow the user to add new components (not implemented yet)
- Go back to the Studio Home and open the search modal
- Find a library v2 component and click on it. It should redirect to the new Library Home.    

---
Private ref: [FAL-3753](https://tasks.opencraft.com/browse/FAL-3753)
